message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -1899,5 +1917,16 @@
       <field type="uint8_t" name="target_system">Fcu target id.</field>
       <field type="uint8_t" name="target_component">Fcu component id, leave 0 by default.</field>
     </message>
+    <message id="12003" name="MNA_CHUTE_STATUS">
+      <description>The send the current stats of the ccdl network on a target.</description>
+      <field type="uint64_t" name="time_usec" units="us">Timestamp (UNIX Epoch time or time since system boot). The receiving end can infer timestamp format (since 1.1.1970 or since system boot) by checking for the magnitude of the number.</field>
+      <field type="uint8_t" name="release_reason" enum="CHUTE_RELEASE_REASON" display="bitmask">Release reason.</field>
+      <field type="uint32_t" name="release_timer_ms" units="ms">Time in ms until the release.</field>
+      <field type="uint8_t" name="standby_state">Standby state (Enabled = True).</field>
+      <field type="int8_t" name="enabled">Enabled (Positive for enabled).</field>
+      <field type="uint8_t" name="release_initiated">Release initiated (0 for no, 1 for release or releasing).</field>
+      <field type="uint8_t" name="target_system">Target id.</field>
+      <field type="uint8_t" name="target_component">Component id, leave 0 by default.</field>
```

Comment:
could we move the target_system and target_component fields to the beginning of the message? to match the other messages